### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by alienlike** [aspnet/Identity#1120]
Before the ASP.Net Core docs were migrated away from RTD, I seem to recall there being some pretty in-depth content on how to implement Identity. Now I'm looking again and the Identity docs contain mostly point-and-click, integration-oriented material. I'm trying to do a custom implementation for Oracle. Is there anything out there for people like me?

**Comment by blowdart** [comment]
Hmm, I don't remember writing anything like that. Could you give me some idea of what you want? I assume you're not wanting to just use EF and take the pain away that way?

**Comment by HaoK** [comment]
Something like this doc? https://docs.microsoft.com/en-us/aspnet/identity/overview/extensibility/overview-of-custom-storage-providers-for-aspnet-identity

**Comment by alienlike** [comment]
Yes, that doc is a good example of what I'm talking about. Most people will use OOB implementations, but there is still a need for custom. In our case we need an Oracle-based implementation consistent with existing data structures, and we don't use EF. We've now implemented what we need, but we muddled through by referring to other custom implementations such as https://github.com/tugberkugurlu/AspNetCore.Identity.MongoDB.

**Comment by huntertran** [comment]
For new comers like me, digging through identity "point-and-click" docs doesn't help much. A lot of questions pop ups, and obviously we cannot post every single questions to Github.
_Why_ you guys created this and that, explanation of this concept, why using this over that, etc. A documentation including all of this would be great.
